Formula/libirecovery.rb

```diff
@@ -0,0 +1,36 @@
+class Libirecovery < Formula
+  desc "Library to communicate with iDevices restore and recovery"
+  homepage "https://www.libimobiledevice.org/"
+
+  stable do
+    url "https://github.com/libimobiledevice/libirecovery/archive/0.1.1.tar.gz"
```

Comment:
Put this with the homepage, stable is the default.

Reply:
Will reach out to @nikias about the website